### ISSUE TITLE
setup.md is now for artemis HPC. Remove and ensure proper rending and flexibility

### ISSUE THREAD

**Issue by dvanic** [Sydney-Informatics-Hub/training-template-old#6]
setup.md is there for Artemis hpc 
- where is it being called for in the lesson?
- how is it rendered?

**Comment by SIHAdmin** [comment]
Almost all of the RC team's lessons require that participants can access Artemis. I'd advise other teams to write their own setup pages, and simply rename them in or out of the build.

setup.md is always built by the template; it is independent of the episodes.

**Comment by dvanic** [comment]
No, it's actually NOT built for a workshop page, and included by default only for lesson pages.
The ones for workshops are in _includes/setup/ which have explicit installation instructions for the different tools. I suggest moving ALL installation/setup instructions in there, and then using the flag system in _config.yml to select which of them will be displayed in the final workshop website. 

 Instead of having to debug multiple instances of

**Comment by SIHAdmin** [comment]
I wonder if this is a non issue, since Artemis courses will probably never be workshops; so by setting the site type to workshop or lesson, the setups from either _includes or [root] will be built accordingly?
